daemon/apparmor_default.go

```diff
@@ -15,14 +15,23 @@ const (
 	defaultAppArmorProfile    = "docker-default"
 )
 
-// DefaultApparmorProfile returns the name of the default apparmor profile
-func DefaultApparmorProfile() string {
+// DefaultAppArmorProfile returns the name of the default apparmor profile
+func DefaultAppArmorProfile() string {
 	if apparmor.HostSupports() {
```

Comment:
For the benefit of other reviewers, I was slightly concerned about this getting invoked in so many places (especially nested, as below), but the underlying implementation uses a `sync.Once` + cached `bool` so it's really efficient. :+1:

https://github.com/containerd/containerd/blob/a545df345ee6b2e0cb6b8ae2c4fd11b7738f9185/pkg/apparmor/apparmor_linux.go#L27-L46